carbon-design-system/carbon-addons-iot-react
# Carbon upgrade 10.48

Closes #3074 

**Summary**

- upgrades carbon to 10.48.0/7.48.1

**Change List (commits, features, bugs, etc)**

- added onClick handler to div wrapping RowActionCells, because the click events for dismissing errors was no longer stopping propagation and was bubbling up to trigger the onRowClicked event. https://github.com/carbon-design-system/carbon/pull/9959
- previously the HotspotEditorDataSourceTab.tests were using `getAllByRole('button')[x]` and selecting an element directly from the array. Some of these calls needed to be changed to add `{name: 'XXX'}` to them, because the number of items with role="button" on the modal had changed.
- remove old styles from the table batch actions to ensure they show correctly with new changes, and remove extra 2px of space (toolbar being 50px instead of 48px tall).
- updated snapshots

**Acceptance Test (how to verify the PR)**

- ensure table row errors dismiss properly
- ensure batch actions appear when items selected

**Regression Test (how to make sure this PR doesn't break old functionality)**

- check stories for other breaks



**Things to look for during review**
